edx_rbac/models.py

```diff
@@ -59,9 +60,19 @@ class UserRoleAssignment(TimeStampedModel, metaclass=UserRoleAssignmentCreator):
     """
     Model for mapping users and their roles.
     """
+    role_class = None
 
     user = models.ForeignKey(settings.AUTH_USER_MODEL, db_index=True, on_delete=models.CASCADE)
-    role_class = None
+
+    applies_to_all = models.BooleanField(
+        default=False,
+        null=False,
+        db_index=True,
```

Comment:
curious why this field should be indexed? maybe i missed the reasoning from the last pr

Reply:
Hmm, yeah, it doesn't need to be.